wormhole-connect/src/utils/coingecko.ts

```diff
@@ -67,10 +67,14 @@ export const fetchTokenMetadata = async (
   tokenId: TokenId,
   params?: CoingeckoParams,
 ): Promise<any> => {
-  return coingeckoRequest(
-    `/api/v3/coins/${tokenId.chain.toLowerCase()}/contract/${tokenId.address.toString()}`,
-    params,
-  );
+  try {
```

Comment:
can we handle this try/catch upstream inside `coingeckoRequest`?

Reply:
Sure